keras/api/wrappers/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""DO NOT EDIT.
+
+This file was autogenerated. Do not edit it by hand,
+since your modifications would be overwritten.
+"""
+
+from keras.src.wrappers._sklearn import KerasClassifier
+from keras.src.wrappers._sklearn import KerasRegressor
+from keras.src.wrappers._sklearn import KerasTransformer
```

Comment:
I have reservations about "KerasTransformer" due to the very specific meaning that Transformer has in deep learning. This is going to confuse the hell out of people who search "Transformer" on keras.io. This was also not part of the original implementation of sklearn wrappers we had in Keras.